packages/ua-devtools/src/oapp/config.ts

```diff
@@ -9,6 +9,7 @@ import {
     formatOmniPoint,
     createConfigureMultiple,
     createConfigureNodes,
+    createConfigureEdges,
 } from '@layerzerolabs/devtools'
```

Comment:
**Local development issue**: When in the `ua-devtools` directory, this package (and a few others in the imports section) is unrecognized in this config file if I install `node_modules` with `pnpm i`. 

If I then delete `node_modules` in `ua-devtools` and use `npm i` to install, they are recognized. I run into this issue with `pnpm i` even after calling `nvm use`.

This _might_ be related to the issue resolved by [this PR](https://github.com/LayerZero-Labs/devtools/pull/631) where we needed to signal to the `Dockerfile` to use the `pnpm` version specified in the `package.json`...

```
ravina@Ravinas-MacBook-Pro ua-devtools % npm -v
9.5.1
ravina@Ravinas-MacBook-Pro ua-devtools % pnpm -v
8.14.0
```